tox.ini

```diff
@@ -23,5 +24,5 @@ commands =
 #     F403 'from ansible.module_utils.basic import *' used; unable to detect undefined names
 #     H303  No wildcard (*) import.
 # Excluding our upstream submodule, and our vendored f5 configuration script.
-ignore=F403,H303
+ignore=F403,H303,R
```

Comment:
Ah, okay that makes sense. I think it would be worth adding something along those lines to the comment above it, ',R' looks more like a typo.

I've tested the patch and it all seems good so other than the above I think it's good to go.

Also, currently gating is broken by https://github.com/rcbops/rpc-openstack/issues/1037 so the checks will fail.